Mathlib/CategoryTheory/Monoidal/Category.lean

```diff
@@ -222,7 +222,79 @@ end MonoidalCategory
 open scoped MonoidalCategory
 open MonoidalCategory
 
-variable (C : Type u) [𝒞 : Category.{v} C] [MonoidalCategory C]
+variable {C : Type u} [𝒞 : Category.{v} C] [MonoidalCategory C]
+
+namespace MonoidalCategory
+
+@[reassoc (attr := simp)]
+theorem hom_inv_whiskerLeft (X : C) {Y Z : C} (f : Y ≅ Z) :
```

Comment:
I may suggest a change:
```suggestion
theorem whiskerLeft_hom_inv (X : C) {Y Z : C} (f : Y ≅ Z) :
```
and similarly for `inv_hom_whiskerLeft`. (But, the `hom_inv_whiskerRight` look good to me.)

Reply:
Thanks! I also renamed the similar lemmas for bicategories for consistency.

Reply:
Thanks! The same change should be done also for `inv_hom_whiskerLeft'` and `hom_inv_whiskerLeft'`.